backend/game/cards/keep_cards/health_manipulation_cards/it_has_a_child.py

```diff
@@ -0,0 +1,21 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+from game.values import constants
+
+
+class ItHasAChild(KeepCard):
+    def __init__(self):
+        Card.__init__(self, "It Has a Child", 7,
+                      "If you reach 0[health] discard all your cards and lose all you [star]. Gain 10[health] and continue playing outside Tokyo",
+                      "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        card = ItHasAChild()
+        player_that_bought_the_card.add_card(card)
+
+    def special_effect(self, player_that_bought_the_card, other_players):
+        if player_that_bought_the_card.current_health == constants.DEATH_HIT_POINT:
```

Comment:
Do we need a constant DEATH_HIT_POINT if it's just a constant for 0? almost seems more confusing since it is very unlikely death would ever occur not at 0.